Lino09/todo-list
# Feat/tests part 2

## Feat/tests part 2

I this PR @Amrhub @carlos-ssh  and I got a test coverage of 100% by adding tests for the:

-  Update check status function
- Update Index function
- Remove completed Tasks function
- Edit tasks
